include/networkit/generators/ChungLuGeneratorAlamEtAl.hpp

```diff
@@ -0,0 +1,64 @@
+/*
+ * ChungLuGeneratorAlamEtAl.hpp
+ *
+ *  Created on: Dec 23, 2013
+ *      Author: Henning
+ *      Contributors: Hoske/Weisbarth/Hering
+ */
+
+#ifndef NETWORKIT_GENERATORS_CHUNG_LU_GENERATOR_ALAM_ET_AL_HPP_
+#define NETWORKIT_GENERATORS_CHUNG_LU_GENERATOR_ALAM_ET_AL_HPP_
+
+#include <networkit/generators/StaticDegreeSequenceGenerator.hpp>
+#include <networkit/generators/StaticGraphGenerator.hpp>
+
+namespace NetworKit {
+
+/**
+ * @ingroup generators
+ * Given an arbitrary degree sequence, the Chung-Lu generative model
+ * will produce a random graph with the same expected degree sequence.
+ *
+ * For reference of the concept of the original model see Chung, Lu: The average distances in random
```

Comment:
`For reference, see the original model by Chung,...`

And use quotes to highlight the paper titles.